tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java

```diff
@@ -154,9 +159,48 @@ private static void configureIfLiferayProject( final IProject project, final Gra
         {
             Job job = new WorkspaceJob( "Checking gradle configuration" )
             {
+
                 @Override
                 public IStatus runInWorkspace( IProgressMonitor monitor ) throws CoreException
                 {
+                    IFile bndFile = project.getFile( "bnd.bnd" );
+
+                    //case 1: has bnd file
+                    if( bndFile != null && bndFile.exists() )
```

Comment:
Oh I see, but lets add those additional checks outside of the Job.  So just move the checks for bnd.bnd and gulpfile outside of the job.  But keep the nature modifications and gradle tooling model checks in the job.

thanks.